pkg/store/bucket.go

```diff
@@ -200,8 +204,9 @@ func newBucketStoreMetrics(reg prometheus.Registerer) *bucketStoreMetrics {
 		Buckets: []float64{0.001, 0.01, 0.1, 0.3, 0.6, 1, 3, 6, 9, 20, 30, 60, 90, 120},
 	})
 	m.resultSeriesCount = promauto.With(reg).NewSummary(prometheus.SummaryOpts{
-		Name: "thanos_bucket_store_series_result_series",
-		Help: "Number of series observed in the final result of a query.",
+		Name:       "thanos_bucket_store_series_result_series",
```

Comment:
This is redundant as the store-api metric will illustrate this data for a single query across all stores, and the store-specific metric you added for `thanos_bucket_store_series_touched` covers the to total fetched for a single storeAPI request.